examples/bq-email-exports/terraform/main.tf

```diff
@@ -0,0 +1,118 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+provider "google" {
```

Comment:
version pin provider

Reply:
Does this mean adding a `version` tag in the provider block?

Reply:
yeah something like 
```tf
provider "google" {
  version = "~> 3.16.0"
}
```